commercetools-models/src/test/java/io/sphere/sdk/subscriptions/SubscriptionFixtures.java

```diff
@@ -165,6 +165,15 @@ public static SubscriptionDraftBuilder snsSubscriptionDraftBuilder(final String
                 .key(snsKey);
     }
 
+    public static SubscriptionDraftBuilder mockPubSubSubscription() {
```

Comment:
How about naming this method `createPubSubSubscription`? I think calling it `mock...` is a bit misleading here, since we really use our draft builder and even send the request to our backend.